src/pages/Login.js

```diff
@@ -16,14 +17,22 @@ class Login extends React.Component {
         name: null,
         email: null,
         password: null,
-        accountId: null
+        accountId: null,
+        shouldRedirect: false
       }
 
       this.updateInput = this.updateInput.bind(this)
       this.onSubmit = this.onSubmit.bind(this)
       this.swapMode = this.swapMode.bind(this)
     }
 
+    componentDidMount() {
+        if (window.sessionStorage.getItem('logout')) {
+            Cookies.remove('accountID')
+            window.sessionStorage.removeItem('logout')
```

Comment:
I don't believe that we need a ```logout``` variable in session storage, nor should we be removing cookies here. The only place time we should remove the account ID cookie should be in the callback function of the logout button, which I imagine will lie in Home.js, not Login.js.